hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpCommandProcessor.java

```diff
@@ -28,6 +28,8 @@
     public static final String URI_CHANGE_CLUSTER_STATE_URL = URI_CLUSTER_MANAGEMENT_BASE_URL + "/changeState";
     public static final String URI_SHUTDOWN_CLUSTER_URL =  URI_CLUSTER_MANAGEMENT_BASE_URL + "/shutdown";
     public static final String URI_FORCESTART_CLUSTER_URL =  URI_CLUSTER_MANAGEMENT_BASE_URL + "/forceStart";
+    public static final String URI_KILLNODE_CLUSTER_URL =  URI_CLUSTER_MANAGEMENT_BASE_URL + "/killNode";
```

Comment:
In lifecycle API we use the terms: shutdown and terminate. Here I would prefer to use `terminate` instead of `killNode` to be coherent with hz api.